src/v2/Components/Artwork/Details.tsx

```diff
@@ -218,6 +234,66 @@ export const Details: React.FC<DetailsProps> = ({
   )
 }
 
+interface LotCloseInfoProps {
+  saleArtwork: any
+  sale: any
```

Comment:
Obvies need to update this. But not totally sure how to use partial types from relay-generated types?

Reply:
One way is to do 
```
Details_artwork["some"]["thing"]
```
But sometimes when it comes to digging into edges, you need to make it non-nullable (i'm not sure why). eg 
```
type SubmissionAsset = NonNullable<UploadPhotos_submission["assets"]>[0]
```

Reply:
oh interesting. thank you!